packages/terra-functional-testing/src/config/wdio.conf.js

```diff
@@ -171,4 +171,8 @@ exports.config = {
     ui: 'bdd',
     timeout: 60000,
   },
+
+  serviceOptions: {
+    selector: '[data-terra-test-content] *:first-child',
```

Comment:
What are your thoughts on defaulting this in the service rather than inline in the config?

Reply:
In that case, do we still keep `serviceOption` but leave it empty? 
https://github.com/cerner/terra-toolkit/pull/464/commits/04718224294139afe8b92e05a16b2910ba550b35